### PR TITLE
User views empty cart#28

### LINKED ISSUE #43: User Story 28, User views their cart show page but it's empty
```
As a visitor or registered user
When I add NO items to my cart yet
And I visit my cart ("/cart")
I see a message that my cart is empty
I do NOT see the link to empty my cart
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #46

depends on #74

### PR DESCRIPTION
**Pull request by danbriechle**
Creates test for registered user viewing an empty cart
Creates test for non registered user viewing an empty cart
Adds if logic to cart show page 

close #43

### CODE REVIEW

File: app/views/carts/show.html.erb [hunk 1]
```diff
@@ -12,4 +10,6 @@
   </div>
 <% end  %>
 <p>Grand Total: <%= number_to_currency(@cart.grand_total/100) %></p>
-<%= link_to "Empty Cart", cart_path %>
+  <% unless @cart_items.count == 0 %>
```
Comment (asmolentzov) [inline app/views/carts/show.html.erb]: I'm not sure, but I think this is the kind of logic we're supposed to avoid in the View, where you're actually asking the view to do some work(`count`). It's not a database call though so I'm not really sure. Either way, I created a `cart_count` method for my user story (it's in master now) that you could use here - `@cart.cart_count`!
